<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDbLoadDataDialog</class>
 <widget class="QDialog" name="CDbLoadDataDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Load new DB data?</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="gb_Entities">
     <property name="title">
      <string>Entities</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="lbl_Load">
        <property name="text">
         <string>New DB data. Load the given entities?</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListView" name="lv_Entities">
        <property name="selectionMode">
         <enum>QAbstractItemView::MultiSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="selectionRectVisible">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wi_WorkStatus" native="true">
     <layout class="QHBoxLayout" name="hl_LoadStatus">
      <item>
       <widget class="QLineEdit" name="le_Info">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="pb_Loading">
        <property name="value">
         <number>50</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wi_Consolidate" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="swift::gui::components::CSimulatorSelector" name="comp_SimulatorSelector">
        <property name="minimumSize">
         <size>
          <width>125</width>
          <height>0</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="hs_Consolidate">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_ModelSet">
        <property name="toolTip">
         <string>Model set</string>
        </property>
        <property name="text">
         <string>Set</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_AllModels">
        <property name="toolTip">
         <string>all models</string>
        </property>
        <property name="text">
         <string>All</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_Consolidate">
        <property name="text">
         <string>consolidate</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="bb_loadDataDialog">
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>swift::gui::components::CSimulatorSelector</class>
   <extends>QFrame</extends>
   <header>gui/components/simulatorselector.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>bb_loadDataDialog</sender>
   <signal>accepted()</signal>
   <receiver>CDbLoadDataDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bb_loadDataDialog</sender>
   <signal>rejected()</signal>
   <receiver>CDbLoadDataDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
